core/LSU.cpp

```diff
@@ -57,20 +58,16 @@ namespace olympia
         // setContinuing to false on any event).
         ldst_pipeline_.setContinuing(true);
 
-        ldst_pipeline_.registerHandlerAtStage<sparta::SchedulingPhase::Update>(static_cast<uint32_t>(PipelineStage::MMU_LOOKUP),
-                                                CREATE_SPARTA_HANDLER(LSU, handleMMULookupReq1_));
-
-        ldst_pipeline_.registerHandlerAtStage<sparta::SchedulingPhase::PostTick>(static_cast<uint32_t>(PipelineStage::MMU_LOOKUP),
-                                              CREATE_SPARTA_HANDLER(LSU, handleMMULookupReq2_));
+        ldst_pipeline_.registerHandlerAtStage<sparta::SchedulingPhase::Update>
+            (static_cast<uint32_t>(PipelineStage::MMU_LOOKUP),
+             CREATE_SPARTA_HANDLER(LSU, handleMMULookupReq_));
 
-        ldst_pipeline_.registerHandlerAtStage<sparta::SchedulingPhase::Update>(static_cast<uint32_t>(PipelineStage::CACHE_LOOKUP),
-                                              CREATE_SPARTA_HANDLER(LSU, handleCacheLookupReq1_));
+        ldst_pipeline_.registerHandlerAtStage<sparta::SchedulingPhase::Update>
```

Comment:
_Might_ be able to keep these in `Tick` phase now.

Reply:
Suraj, noticed that you changed this in the other PR.  Can you change it here too (and test)?